README.md

```diff
@@ -17,18 +17,15 @@ The [official CocoaPods website](http://cocoapods.org) has detailed instructions
 The following lines in a Podfile will install the SDK for an iOS app with a watch OS 2 extension: 
 
 ```
-source 'https://github.com/openwhisk/openwhisk-podspecs.git'
-
+install! 'cocoapods', :deterministic_uuids => false
 use_frameworks!
 
 target 'MyApp' do
-     platform :ios, '9.0'
      pod 'OpenWhisk'
 end
 
 target 'MyApp WatchKit Extension' do 
-     platform :watchos, '2.0'
-     pod 'OpenWhiskWatch'
+     pod 'OpenWhisk'
```

Comment:
If you do not intend to publish to Cocoapods master Specs repo, then yes that line will be necessary. Alternatively, we can put back the `source 'https://github.com/openwhisk/openwhisk-podspecs.git'` line at the top. I think your suggestion would be better since that `source` line seems to cause confusion with users unfamiliar with Cocoapods.